SofaKernel/modules/SofaExplicitOdeSolver/EulerSolver.cpp

```diff
@@ -46,92 +46,132 @@ using namespace sofa::defaulttype;
 using namespace sofa::helper;
 using namespace core::behavior;
 
-int EulerSolverClass = core::RegisterObject("A simple explicit time integrator")
-        .add< EulerSolver >()
+int EulerExplicitSolverClass = core::RegisterObject("A simple explicit time integrator")
+        .add< EulerExplicitSolver >()
         .addAlias("Euler")
         .addAlias("EulerExplicit")
         .addAlias("ExplicitEuler")
-        .addAlias("EulerExplicitSolver")
+        .addAlias("EulerSolver")
         .addAlias("ExplicitEulerSolver")
         ;
 
-EulerSolver::EulerSolver()
-    : symplectic( initData( &symplectic, true, "symplectic", "If true, the velocities are updated before the positions and the method is symplectic (more robust). If false, the positions are updated before the velocities (standard Euler, less robust).") )
+EulerExplicitSolver::EulerExplicitSolver()
+    : d_symplectic( initData( &d_symplectic, true, "symplectic", "If true, the velocities are updated before the positions and the method is symplectic (more robust). If false, the positions are updated before the velocities (standard Euler, less robust).") )
```

Comment:
If one is provably less robust why do we use it ?

Reply:
Hi Hugo, 

Thanks for your answer but ... 

I still wonder if it make good sense to increase the Sofa complexity both in term of code (readability, code maintenance costs, needs more unittests to validate the features) as well as  user interface for something that is *provably* bad and for which we have a good solution ?

Of course if in some case the non-simplectic version is better in anyways then the picture would be different but this is not the case isn't it ?